data/human/deep missions.txt

```diff
@@ -1398,89 +1415,168 @@ mission "Deep: Remnant 1A"
 	to offer
-		has "Deep: Remnant 1A: done"
-	
+		or
+			has "Deep: Remnant A: done"
+			and
+				has "Deep: Remnant: Keystone Research: done"
+				not "Deep: Remnant A: offered"
+				not "Deep: Remnant 1A: offered"
+				not "Deep: Remnant 1B: offered"
+				not "Deep: Remnant 1C: offered"
+				or
+					has "deep: knew of stones before scientists"
+					has "First Contact: Remnant: offered"
 	on offer
 		conversation "deep: remnant"
+	on visit
+		dialog `You've returned to <planet>, but you haven't traveled through all the wormholes yet. Return to the strange region of space and travel through the loop of wormholes.`
+
+
+
+conversation "deep: remnant"
+	branch "on second mission" "on first mission"
+		has "Deep: Remnant A: done"
 	
-	npc "scan outfits"
-		government "Remnant"
-		personality staying uninterested target
-		system "Arculus"
-		fleet
-			names "remnant"
-			variant
-				"Albatross"
-				"Starling"
-		dialog `You have finished scanning the Remnant ships in the system. You may now return to <destination> with the scanner logs.`
+	label "on first mission"
+	`You contact Ivan after you land and he sends you directions to his research lab, located on the outskirts of the city. When you arrive, you hardly have time to greet Ivan, Rayna, and the others before Ivan begins discussing what they have learned from the Quantum Keystones.`
+	branch "knew before" "didn't know before"
+		has "deep: knew of stones before scientists"
 	
-	on visit
-		dialog `You show your scanner logs to Ivan, but he shakes his head. "This isn't enough. There should be more ships for you to scan in <waypoints>. Make sure you're using an outfit scanner as well."`
-	on complete
-		event "deep: scan log research" 14
-		log `Scanned Remnant ships beyond the wormhole in Terminus and handed the scan logs over to Ivan and his team.`
-		dialog
-			`The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says. "Then we will contact you about what we want you to do next."`
+	label "knew before"
+	`	"As you mentioned before, a single Keystone is sufficient to enter the wormhole," Ivan says. "We've determined that there is a stabilization effect, either on the ship or the wormhole, that allows matter to cross the threshold. However, we'll need much more time and data to fully understand the true mechanism.`
+	branch remnant1
+		has "deep: did reveal remnant"
+	`	"Last time, you mentioned that there were alien creatures able to survive the vacumm of space, correct?"`
+	`	You nod.`
+	`	"Right. Well, it would be extremely interesting if we were to learn more about these alien creatures.`
+		goto end
+
+	label remnant1
+	`	"Last time, you mentioned that there were humans living in this space, the... the 'Remainder?'"`
+	`	"Remnant," you say.`
+	`	"Right, that was it. Well, it would be extremely interesting if we were to learn more about this people.`
```

Comment:
Shouldn't this read ```"Right, that was it. Well, it would be extremely interesting if we were to learn more about these people.```

Reply:
When "people" refers to a society/collective group rather than a bunch of individuals, I think it can be treated as singular instead of plural.

Reply:
In conversation though?  Also, missing a double quote at the end (in case that wasn't obvious).

Reply:
Don't need a quotation mark; the speaker continues in the next line after this (`label end`). It's a small thing, but it's rule of thumb for multiple paragraphs with one speaker to have no ending quotation mark until the quotation is entirely completed. As far as whether "people" is right, I agree it does sound strange if you're not used to hearing it (it's not very common), but I don't think I've ever heard that it's wrong in American English. Could you link something that backs that point up?

Reply:
Apologies, I didn't realize the conversation continued.  But then shouldn't the starting quote in the end conversation not be there or is that to clarify continuing speaking.

Sadly I cannot.  Everything I've found states using the singular "this" is correct.  Many ignore this for the same reason I do but it is not official in any way.

Reply:
Your second take is right; [here's an article on it](http://baxtercommunications.nl/quotation-marks-multi-paragraph-rule/#.X5XyJFMpAwA).
The singular "people" is a strange case. English is strange in general, I guess.

Reply:
> English is strange in general, I guess.

Yeah, English is definitely strange. Even worse when we've got multiple contradictory variants of it that don't *sound* contradictory, but definitely *look* it.